src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -114,12 +136,27 @@ class ExplorerButtonGroup extends React.Component {
     if (this.props.filter.data_format) {
       filter.data_format = this.props.filter.data_format;
     }
-    let resultManifest = await this.props.downloadRawDataByTypeAndFilter(
-      resourceType, filter, [
-        refFieldInResourceIndex,
-        resourceFieldInResourceIndex,
-      ],
-    );
+    let resultManifest;
+    try {
+      resultManifest = await this.props.downloadRawDataByTypeAndFilter(
+        resourceType,
+        filter,
+        [
+          refFieldInResourceIndex,
+          resourceFieldInResourceIndex,
+          ...additionalFields,
```

Comment:
@mfshao Something @svburke brought up, will the extra fields be a breaking change for gen3-client? https://github.com/uc-cdis/cdis-data-client/blob/531982ea94edb98cbd706ab6656e4a7d790690e9/gen3-client/g3cmd/download-multiple.go#L336

Reply:
Update: No, it shouldn't  -- plus, I tested it and gen3-client knew what to do with the new manifests.